website/src/components/SearchPage/DownloadDialog/LinkOutMenu.tsx

```diff
@@ -0,0 +1,156 @@
+                    onClick={() => setIsOpen(!isOpen)}
+                >
+                    <span>Tools</span>
+                    <IwwaArrowDown className='ml-2 h-5 w-5' aria-hidden='true' />
+                </MenuButton>
+
+                <MenuItems className='absolute right-0 mt-2 w-56 origin-top-right rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none'>
+                    <div className='py-1'>
+                        {linkOuts.map((linkOut) => (
+                            <MenuItem key={linkOut.name}>
+                                {({ focus }) => (
+                                    <button
+                                        onClick={() => handleLinkClick(linkOut)}
+                                        className={`
+                                            ${focus ? 'bg-gray-100 text-gray-900' : 'text-gray-700'}
+                                            flex items-center justify-between px-4 py-2 text-sm w-full text-left
+                                        `}
+                                    >
+                                        {linkOut.name}
+                                        <DashiconsExternal className='h-4 w-4 ml-2' />
+                                    </button>
+                                )}
+                            </MenuItem>
+                        ))}
+                    </div>
+                </MenuItems>
+            </Menu>
+
+            <BasicModal isModalVisible={isModalVisible} setModalVisible={setModalVisible}>
+                <div className='p-6'>
+                    <h2 className='text-xl font-bold mb-2'>
+                        Options for launching {currentLinkOut.current?.name ?? 'Tool'}
+                    </h2>
+                    <h3 className='text-lg font-medium text-gray-700 mb-4 mt-6'>Data use terms</h3>
+                    <p className='mb-6 text-gray-600'>
+                        Would you like to include restricted-use sequences in this analysis? (If you do, you must comply
+                        with the Restricted-Use terms.)
+                    </p>
+                    <div className='flex justify-end space-x-4'>
+                        <button
+                            className='px-4 py-2 border border-gray-300 rounded-md text-gray-700 font-medium hover:bg-gray-50 transition-colors'
+                            onClick={handleOpenLinkWithOpenOnly}
+                        >
+                            Open sequences only
+                        </button>
+                        <button
+                            className='px-4 py-2 bg-primary-600 text-white rounded-md font-medium hover:bg-primary-700 transition-colors'
+                            onClick={handleIncludeRestricted}
+                        >
+                            Include Restricted-Use
+                        </button>
+                    </div>
+                </div>
```

Comment:
This should only be visible if the data use terms are enabled, right?

Reply:
Ideally yes I guess: for now this would only properly support the case where they are. I made an issue for this: https://github.com/loculus-project/loculus/issues/4114, which from my side is relatively blocked on https://github.com/loculus-project/loculus/issues/4113 .

Reply:
Why would this be blocked by #4113? We already have instances that disable the data use terms and it would make a lot of sense to use this feature for and https://loculus.genspectrum.org/. Is there any particular difficulty to adapt the code to disable the modal when data use terms are not used?

Reply:
(If you don't have the time to change it at the moment, I can give it a stab tomorrow.)

Reply:
It's blocked in the sense that there is no easy way for me to see if it's working - of course very happy for anyone to take a stab at it! But I don't think it needs to block merging - in the same way that the file sharing feature isn't ready to use on any instance but we're going to merge it.